networking_private_dns_resolver_inbound_endpoints.tf

```
@@ -0,0 +1,28 @@
+
+module "private_dns_resolver_inbound_endpoints" {
+  source   = "./modules/networking/private_dns_resolvers_inbound_endpoints"
+  for_each = local.networking.private_dns_resolver_inbound_endpoints
+
+  global_settings         = local.global_settings
+  client_config           = local.client_config
+  settings                = each.value
+  private_dns_resolver_id = can(each.value.private_dns_resolver_id) ? each.value.private_dns_resolver_id : local.combined_objects_private_dns_resolvers[try(each.value.private_dns_resolver.lz_key, local.client_config.landingzone_key)][each.value.private_dns_resolver.key].id
+  inherit_tags            = local.global_settings.inherit_tags
+  location                = try(local.global_settings.regions[each.value.region], each.value.region)
```

If the private dns resolver id is set we need the region to be set to get the location.
If not set then we pick it up from the private dns resolver
do the same for base tags
```suggestion
  location                = can(each.value.private_dns_resolver_id) || can(each.value.private_dns_resolver.id) || can(each.value.region) ? local.global_settings.regions[each.value.region] : local.combined_objects_private_dns_resolvers[try(each.value.private_dns_resolver.lz_key, local.client_config.landingzone_key)][each.value.private_dns_resolver.key].location
  tags = can(each.value.private_dns_resolver_id) || can(each.value.private_dns_resolver.id) || local.global_settings.inherit_tags == false ? null : try(local.combined_objects_private_dns_resolvers[try(each.value.private_dns_resolver.lz_key, local.client_config.landingzone_key)][each.value.private_dns_resolver.key].tags, {})
```